diff -ruN cosmic-greeter/cosmic-greeter.toml cosmic-greeter-patched/cosmic-greeter.toml
--- cosmic-greeter/cosmic-greeter.toml	2025-09-30 00:32:00.541349933 +0300
+++ cosmic-greeter-patched/cosmic-greeter.toml	2025-09-30 23:52:43.663248619 +0300
@@ -6,4 +6,4 @@
 
 [default_session]
 command = "cosmic-greeter-start"
-user = "cosmic-greeter"
+user = "greeter"
diff -ruN cosmic-greeter/debian/cosmic-greeter.tmpfiles cosmic-greeter-patched/debian/cosmic-greeter.tmpfiles
--- cosmic-greeter/debian/cosmic-greeter.tmpfiles	2025-09-30 00:32:00.541895409 +0300
+++ cosmic-greeter-patched/debian/cosmic-greeter.tmpfiles	2025-09-30 23:57:17.434918590 +0300
@@ -1,3 +1,3 @@
 # Home directory of cosmic-greeter
-d	/var/lib/cosmic-greeter	0750	cosmic-greeter	cosmic-greeter
-d	/run/cosmic-greeter	    0755	cosmic-greeter	cosmic-greeter	-
+d	/var/lib/cosmic-greeter	0750	greeter	greeter
+d	/run/cosmic-greeter	    0755	greeter	greeter	-
diff -ruN cosmic-greeter/src/main.rs cosmic-greeter-patched/src/main.rs
--- cosmic-greeter/src/main.rs	2025-09-30 00:32:00.545597098 +0300
+++ cosmic-greeter-patched/src/main.rs	2025-09-30 23:55:05.658846814 +0300
@@ -30,7 +30,7 @@
 
     match pwd::Passwd::current_user() {
         Some(current_user) => match current_user.name.as_str() {
-            "cosmic-greeter" => greeter::main(),
+            "greeter" => greeter::main(),
             _ => locker::main(current_user),
         },
         _ => Err("failed to determine current user".into()),
