diff -rub cosmic-greeter/cosmic-greeter.toml cosmic-greeter-patched/cosmic-greeter.toml
--- cosmic-greeter/cosmic-greeter.toml	2025-04-24 19:34:03.563449800 +0200
+++ cosmic-greeter-patched/cosmic-greeter.toml	2025-04-24 19:42:01.295997824 +0200
@@ -6,4 +6,4 @@
 
 [default_session]
 command = "cosmic-comp systemd-cat -t cosmic-greeter cosmic-greeter"
-user = "cosmic-greeter"
+user = "greeter"
diff -rub cosmic-greeter/debian/cosmic-greeter.tmpfiles cosmic-greeter-patched/debian/cosmic-greeter.tmpfiles
--- cosmic-greeter/debian/cosmic-greeter.tmpfiles	2025-04-24 19:34:03.564332630 +0200
+++ cosmic-greeter-patched/debian/cosmic-greeter.tmpfiles	2025-04-24 19:42:01.296273906 +0200
@@ -1,2 +1,2 @@
 # Home directory of cosmic-greeter
-d	/var/lib/cosmic-greeter	0750	cosmic-greeter	cosmic-greeter
\ No newline at end of file
+d	/var/lib/cosmic-greeter	0750	greeter	greeter
diff -rub cosmic-greeter/src/main.rs cosmic-greeter-patched/src/main.rs
--- cosmic-greeter/src/main.rs	2025-04-24 19:34:03.697333204 +0200
+++ cosmic-greeter-patched/src/main.rs	2025-04-24 19:42:01.296509138 +0200
@@ -30,7 +30,7 @@
 
     match pwd::Passwd::current_user() {
         Some(current_user) => match current_user.name.as_str() {
-            "cosmic-greeter" => greeter::main(),
+            "greeter" => greeter::main(),
             _ => locker::main(current_user),
         },
         _ => Err("failed to determine current user".into()),
Only in cosmic-greeter-patched/src: main.rs.orig
